tests/conftest.py

```diff
@@ -32,6 +32,7 @@ def version(request, host):
             'source %s && echo $VERSION', str(product_path)
         )
 
+
 @pytest.fixture(scope="module")
 def hostname(host):
```

Comment:
We'd need to rename this fixture, since it's the minion ID, not hostname... For later.

Reply:
Nop, this one return the real hostname, not the minion ID you have another one just bellow `nodename` to retrieve the minion ID (but maybe this "hostname" one is no longer needed)

Reply:
:rofl: got tricked by the collapsed lines :facepalm: please ignore!